core/exception_vectors.S

```diff
@@ -490,46 +500,28 @@ UserExceptionHandler:
         wsr     a0, ps
         rsync
         rsr     a2, exccause
-        beqi    a2, CAUSE_LVL1INT, UserHandleInterrupt
-        /* Any UserException cause other than level 1 interrupt should panic */
-UserFailOtherExceptionCause:
-        break   1, 1
-        call0   sdk_user_fatal_exception_handler
-UserHandleInterrupt:
+        /* Any UserException cause other than a level 1 interrupt is fatal */
+        bnei    a2, CAUSE_LVL1INT, .UserFailOtherExceptionCause
+.UserHandleInterrupt:
```

Comment:
This label should probably be .LUserHandleInterrupt (assuming it was intended to be a local label)